app/initializers/metrics.js

```diff
@@ -1,16 +0,0 @@
-import config from '../config/environment';
-
-export function initialize() {
-  const application = arguments[1] || arguments[0];
-  const { metricsAdapters = [] } = config;
-  const { environment = 'development' } = config;
-  const options = { metricsAdapters, environment };
-
-  application.register('config:metrics', options, { instantiate: false });
-  application.inject('service:metrics', 'options', 'config:metrics');
```

Comment:
isn't removing this a breaking change?

Reply:
deleting the initializer? That is what the entire PR is all about. moving that from an implicit injection to an explicit one.  This code is put on the metrics service init.   see https://github.com/emberjs/rfcs/pull/680 and https://deprecations.emberjs.com/v3.x/#toc_implicit-injections


Given that the service named metrics in this addon will get the end result (see service init method) the only time I could think this would be a breaking change would be if someone created a service named metrics and did inherit from the service in this addon but still expected the property to be there on their service. Im not sure thats a reasonable thing.

Reply:
right, sorry, I totally misread what the code was doing 😅